src/modules/services/main.py

```diff
@@ -35,37 +38,64 @@ def run():
 
     # enable services
     for svc in services:
-        ec = libcalamares.utils.target_env_call(['systemctl', 'enable', '{}.service'.format(svc['name'])])
+        ec = libcalamares.utils.target_env_call(
+            ['systemctl', 'enable', '{}.service'.format(svc['name'])]
+            )
 
         if ec != 0:
             if svc['mandatory']:
-                return "Cannot enable systemd service {}".format(svc['name']), \
-                       "systemctl enable call in chroot returned error code {}".format(ec)
+                return ("Cannot enable systemd service {}".format(svc['name']),
+                        "systemctl enable call in chroot returned error code",
+                        "{}".format(ec)
+                        )
             else:
-                libcalamares.utils.debug("Cannot enable systemd service {}".format(svc['name']))
-                libcalamares.utils.debug("systemctl enable call in chroot returned error code {}".format(ec))
+                libcalamares.utils.debug(
+                    "Cannot enable systemd service {}".format(svc['name'])
+                    )
+                libcalamares.utils.debug(
+                    "systemctl enable call in chroot returned error code "
+                    + "{}".format(ec)
+                    )
 
     # enable targets
     for tgt in targets:
-        ec = libcalamares.utils.target_env_call(['systemctl', 'enable', '{}.target'.format(tgt['name'])])
+        ec = libcalamares.utils.target_env_call(
+            ['systemctl', 'enable', '{}.target'.format(tgt['name'])]
+            )
 
         if ec != 0:
             if tgt['mandatory']:
-                return "Cannot enable systemd target {}".format(tgt['name']), \
-                       "systemctl enable call in chroot returned error code {}".format(ec)
+                return ("Cannot enable systemd target {}".format(tgt['name']),
+                        "systemctl enable call in chroot returned error code",
```

Comment:
Remove comma. Insert space before closing quote -- then you've got multiple string literals that will format correctly, and you don't change it to a 3-tuple.